.github/workflows/docs/build-docs

```diff
@@ -75,8 +75,6 @@ def build_module_docs():
             "-D",
             f"project=pytket-{MODULE}",
             "-D",
-            f"copyright={datetime.date.today().year} Cambridge Quantum Computing",
```

Comment:
Why remove, and not just update to Quantinuum?

Reply:
We are not publishing the docs from this repo anymore, what we would set here would stop us from setting it later at a central point for all extensions.